### DISCUSSION TITLE
useNuxtApp or useI18n in server route / api

### DISCUSSION THREAD

**Discussion by mlankamp** [nuxt/nuxt#19389] [Questions] [answered]
Is it possible to get hold on the i18n or nuxt instance in a server call (defineEventHandler)? I'm asking because I want to do some translation in an API call

**Comment by TheAlexLichter** [comment, marked as answer]
Not really, because Nitro will handle the call *before* the nuxt instance will.

**Reply by mlankamp** [reply]
Thanks
